src/RecordManager/Finna/Record/MimeTypeTrait.php

```diff
@@ -70,7 +62,29 @@ trait MimeTypeTrait
         'medium',
         'square',
         'thumb',
-        'zoomview',
+        'zoomview'
+    ];
+
+    /**
+     * These values are not to be checked from a link, as they
+     * can return a false mime type.
```

Comment:
'a false mime type' => 'an invalid mime type (e.g. an address ending in .php is probably executed server-side and could return anything)'